obstore/python/obstore/store/_aws.pyi

```diff
@@ -544,8 +546,9 @@ class S3Store:
     def from_session(
         cls,
         session: boto3.session.Session | botocore.session.Session,
-        bucket: str,
+        bucket: str | None = None,
         *,
+        prefix: str | None = None,
```

Comment:
same questions as above about whether this change needs to be copied to the other store classes?